tlsfuzzer/messages.py

```diff
@@ -364,9 +364,17 @@ def process(self, state):
 
 class PlaintextMessageGenerator(Command):
     """
-    Send a plaintext data record even if encryption is already negotiated.
+    Send a plaintext data record irrespective of encryption state.
```

Comment:
I am partial to "regardless" but irrespective is not wrong, just harder to say

Reply:
thankfully, one just needs to be able to read it :smiley: